src/pages/_app.page.tsx

```diff
@@ -1,16 +1,103 @@
+import { Layout, Modal, Space } from 'antd'
+import { Content } from 'antd/lib/layout/layout'
+import { useRouter } from 'next/router'
+import Navbar from 'components/Navbar'
+import { NetworkContext } from 'contexts/networkContext'
+import useMobile from 'hooks/Mobile'
+import { NetworkName } from 'models/network-name'
 import type { AppProps } from 'next/app'
-import React from 'react'
-import { Head } from 'components/common'
+import React, { useContext, useEffect, useState } from 'react'
+import { CoreAppWrapper, Head } from 'components/common'
+
+import { readNetwork } from 'constants/networks'
 
 import '../styles/antd.css'
 import '../styles/index.scss'
 
+// TODO: Move this to each page where needed.
```

Comment:
I think moving this back will lose our SEO changes, but if it means site is reliable, probably a good call

Reply:
Yep it will :( all g we'll get it back quick. Lots of the site is breaking atm tho